jekyll-assets/_layouts/docs.html

```diff
@@ -1,25 +1,181 @@
+                        </a>
+                      </a>
+                    </div>
+                  </label>
+                </span>
+              </li>
+              {% for item in subdir.toc %}
+              <div class="itemcontents">
+                <ul class="sectlevel1">
+                  <input class="toc-toggle-box" type="checkbox" id="{{ item.path }}"
+                    {% if page.url contains item.path %}
+                    checked=true
+                    {% endif %}
+                  />
+                  <li>
+                    <span>
+                      <label class="toc-toggle" for="{{ item.path }}">
+                        <div class="toc-toggle-container">
+                          <a class="level2" href="{{ site.baseurl }}{{ item.path }}">
+                            <a alt="expand documentation category" title="expand documentation category"
+                              {% if page.url contains item.path %}
+                              class="bold toc-item"
+                              {% else %}
+                              class="toc-item" 
+                              {% endif %}
+                            >
+                            {{ item.title | markdownify }}
+                            </a>
+                          </a>
+                        </div>
+                      </label>
+                    </span>
+                  </li>
+                  {% if item.sections[0] %}
+                  {% for entry in item.sections %}
+                  <div class="itemcontents">
+                    <ul class="sectlevel1">
+                      <input class="toc-toggle-box" type="checkbox" id="{{ item.path }}#{{ entry.anchor }}"
+                        {% if endcapture contains entry.anchor %}
+                        checked=true
+                        {% endif %}
+                      />
+                      <li>
+                        <span>
+                          <label class="toc-toggle"
+                            {% if entry.subsections[0] %}
+                            for="{{ item.path }}#{{ entry.anchor }}"
+                            {% else %}
+                            {% endif %}>
+                            <div class="toc-toggle-container">
+                              {% if entry.subsections[0] %}
+                              <a class="level3" href="{{ site.baseurl }}{{ item.path }}#{{ entry.anchor }}" >
+                                <a id="{{ item.path }}#{{entry.anchor}}-toc" class="toc-item" alt="expand documentation category" title="expand documentation category"
+                                >
+                                <style onload="if (window.location.hash.includes('{{ entry.anchor }}')) { document.getElementById('{{ item.path }}#{{entry.anchor}}-toc').setAttribute('style', 'font-weight:bold'); document.getElementById('{{ item.path }}#{{ entry.anchor }}').checked = true; }"></style>
```

Comment:
Does this bit of JS belong only here or could it be extracted elsewhere given the associated `style` element is empty?